dataloader.go

```diff
@@ -20,12 +21,17 @@ import (
 // web request.
 type Interface interface {
 	Load(string) Thunk
+	LoadContext(context.Context, string) Thunk
```

Comment:
I like that this is non-breaking. I've seen projects like [goji]() do something like this before go 1.7 put the context on `http.Request`.

They used the convention of shortening context to `C`, e.g. `LoadC(context.Context, string)`, `LoadManyC(context.Context, []string)`, `type BatchFuncC` etc.

What do you think about adopting that naming convention?